mmdeploy/pytorch/ops/adaptive_avg_pool.py

```diff
@@ -78,3 +79,30 @@ def adaptive_avg_pool3d__default(ctx, *args):
     Align symbolic of adaptive_pool between different torch version.
     """
     return adaptive_avg_pool3d(*args)
+
+
```

Comment:
Why don't we use 
```python
@SYMBOLIC_REWRITER.register_symbolic('adaptive_avg_pool2d', backend='ncnn', is_pytorch=True)
```
just like the symbolic above?
We can use `torch.nn.functional.adaptive_avg_pool2d` instead of `NcnnAdaptiveAvgPoolingOp.apply`